zone/effects.cpp

```diff
@@ -793,6 +793,8 @@ void EntityList::AESpell(
 	}
 	else if (IsTargetableAESpell(spell_id) && is_detrimental_spell && !is_npc) {
 		max_targets_allowed = 4;
+		if (spells[spell_id].effectid[2] == 18 || spells[spell_id].effectid[0] == 31) // pacify and mez
```

Comment:
Please use constants here for 18 and 31

`SE_Lull`
`SE_Mez`

From `spdat.h`

Also others who know more than me will need to comment on the Live like relevance of this fix

Reply:
constants added on commit after this

as for it being valid live, Drogerin checked on live using a enchanter to make sure there wasn't a cap for mez and pacify
![image](https://user-images.githubusercontent.com/483540/118061806-4e2e8700-b35b-11eb-8ff4-48e1d608efb0.png)

Reply:
Is there a reason we're restricting the checking here to specifically effect slots 2 and 0 selectively for each type ?

Reply:
pacify is only on slot 2 and mez is only on slot 1, we could check each slot for both types i guess but this seemed the most efficient and costing the least amount of time since we know what slots those effects will always be at